diesel/src/migrations/mod.rs

```diff
@@ -145,10 +144,9 @@ pub fn create_schema_migrations_table_if_needed<Conn: Connection>(conn: &Conn) -
     })
 }
 
-fn previously_run_migration_versions<Conn: Connection>(conn: &Conn) -> QueryResult<HashSet<String>> {
+fn previously_run_migration_versions<Conn: Connection>(conn: &Conn) -> QueryResult<Vec<String>> {
     __diesel_schema_migrations.select(version)
         .load(conn)
-        .map(|r| r.collect())
```

Comment:
This should still return a `HashSet` regardless. This line should be `.map(HashSet::from_iter)`